FinancialTracker/ViewControllers/BalanceViewController.swift

```diff
@@ -85,6 +85,7 @@ class BalanceViewController: UIViewController {
                 
                 guard let authManager = self.authManager else { return }
                 
+                tabBarVC.justMadeAccount()
                 tabBarVC.setAuthManager(authManager)
```

Comment:
Looks like that if you have called `tabBarVC.setAuthManager(authManager)`, there is no need to call `tabBarVC.justMadeAccount()` too.